### DISCUSSION TITLE
How to get the Development Seelction ?

### DISCUSSION THREAD

**Discussion by Noschvie** [benzino77/tasmocompiler#354] [Q&A] [answered]
How can I get the development selection to be compiled ?
Thanks!

![grafik](https://github.com/benzino77/tasmocompiler/assets/48217146/3531fd2a-80d1-4eb4-8cb9-6eb2d8868ad6)

... thanks for this great tool!

**Comment by benzino77** [comment, marked as answer]
https://github.com/benzino77/tasmocompiler#development-version
